docs/base.md

```diff
@@ -212,8 +213,50 @@ end
 
 ---
 
+####spawnObjectData(...)
 
+Spawns an Object using a data table. Works with [getData()](object.md#getdata). It works just like spawnObject, but instead of a `type`, you supply a `data` table. The other parameters will overwrite those in the Data.
 
+!!!tip
+	Spawned Objects take a moment to be physically spawned into the game. The purpose of the callback functionality is to allow you to run additional actions after the Object has been initiated fully into the instance. You can also add a delay after spawning using a [Wait](wait.md) function.
+
+!!!info "spawnObjectData(parameters)"
+	* [<span class="tag tab"></span>](types.md) **parameters**: A Table of parameters used to determine how spawnObjectData will act.
+		* [<span class="tag tab"></span>](types.md) **parameters.data**: [getData()](object.md#getdata) string.
```

Comment:
Probably need to remove string from the end of this line; possibly replace with table?

Reply:
You're right, good catch.  I've added a new commit to address that, thanks.